### PR TITLE
Ability to handle import logs from multiple projects

### PR DESCRIPTION
**Pull request by vzhukovs**
### What does this PR do?
Add ability to transmit importer logs from multiple project at the same time.

### What issues does this PR fix or reference?
#5428 

#### Changelog
Add ability to transmit importer logs from multiple project at the same time

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/ProjectImportNotificationSubscriber.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+public class ProjectImportNotificationSubscriber {
+
+    public static final String WS_AGENT_ENDPOINT = "ws-agent";
+
+    @Inject
+    public ProjectImportNotificationSubscriber(EventBus eventBus, RequestTransmitter transmitter) {
+        eventBus.addHandler(WsAgentStateEvent.TYPE, new WsAgentStateHandler() {
+            @Override
+            public void onWsAgentStarted(WsAgentStateEvent event) {
+                transmitter.newRequest().endpointId(WS_AGENT_ENDPOINT).methodName(EVENT_IMPORT_OUTPUT_SUBSCRIBE).noParams().sendAndSkipResult();
```
Comment (vparfonov) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/ProjectImportNotificationSubscriber.java]: formatting issue

**Review by vparfonov — APPROVED** [review]

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/ProjectImportNotificationSubscriber.java (hunk 1 above)
Comment (vzhukovs) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/ProjectImportNotificationSubscriber.java]: Fixed

**Review by dkulieshov — APPROVED** [review]

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/ProjectImportNotificationSubscriber.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+/**
+ * Subscriber that register and deregister a listener for import project progress.
+ *
+ * @author Vlad Zhukovskyi
+ */
+@Singleton
+public class ProjectImportNotificationSubscriber {
+
+    public static final String WS_AGENT_ENDPOINT = "ws-agent";
```
Comment (ashumilova) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/ProjectImportNotificationSubscriber.java:32]: may this property be defined at one place similar to AppContext one?

**Review by ashumilova — APPROVED** [review]

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/ProjectImportNotificationSubscriber.java (hunk 2 above)
Comment (vzhukovs) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/ProjectImportNotificationSubscriber.java:32]: it's an endpoint id which is not stored in application context

**Pull request merged**